app/javascript/dashboard/routes/dashboard/settings/personal/MessageSignature.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="flex flex-col gap-6">
+    <form class="flex flex-col gap-6" @submit.prevent="updateSignature()">
```

Comment:
I don't think we need a `div` wrapping the `form`, since there's no other children